src/Microsoft.ML.Transforms/WhiteningTransform.cs

```diff
@@ -185,15 +215,19 @@ public void Save(ModelSaveContext ctx)
 
         private const Mkl.Layout Layout = Mkl.Layout.RowMajor;
 
-        // Stores whitening matrix as Float[] for each column.
-        private readonly Float[][] _models;
-        // Stores inverse ("recover") matrix as Float[] for each column. Temporarily internal as it's used in unit test.
+        // Stores whitening matrix as float[] for each column.
+        private readonly float[][] _models;
+        // Stores inverse ("recover") matrix as float[] for each column. Temporarily internal as it's used in unit test.
         // REVIEW: It doesn't look like this is used by non-test code. Should it be saved at all?
```

Comment:
Maybe add "_models[i] is the whitening matrix of the i-th column." can make the memory layout clearer?